pydis_site/urls.py

```diff
@@ -2,5 +2,5 @@
 
 
 urlpatterns = (
-    path('', include('pydis_site.apps.home.urls', namespace='home')),
+    path('', include('pydis_site.apps.main.urls', namespace='home')),
```

Comment:
this is a bit contradictory because the app name is `main` but you namespace it under `home`